.github/workflows/nix-jupyterlab.yml

```diff
@@ -23,4 +23,4 @@ jobs:
         signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
     - run: |
         cachix use iohk
-        nix-build nix/jupyter-shell.nix -j 1 --option sandbox false 
+        nix-shell nix/jupyter-shell.nix -j 1 --option sandbox false --command echo
```

Comment:
that should be left at `nix-build`. the intention here is to build the shell derivation and cache it.